CITATION.cff

```diff
@@ -0,0 +1,27 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+version: 1.8.1
```

Comment:
I don't think we shou;d hard-code the version number here, it would have to be updated with every release. Just add something to the `message` saying that the version used should be included in the citation?

Reply:
Good idea. Thanks.